app/views/propertyrepresentation/manage/addAgentToAllProperty.scala.html

```diff
@@ -0,0 +1,53 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.propertyrepresentation.ManageAgentOptionItem
+@import models.propertyrepresentation.ManageAgentOptions
+@import models.propertyrepresentation.RemoveFromYourAccount
+@import models.propertyrepresentation.ManageAgentRequest
+@import models.propertyrepresentation.AgentSummary
+@import models.propertyrepresentation.AppointmentScope.ALL_PROPERTIES
+@import models.propertyrepresentation.AppointAgentRequest
+
+@import config.ApplicationConfig
+
+@this(govukErrorSummary : GovukErrorSummary, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout)
+
+
+@(form: Form[AppointAgentRequest], agentName: String, agentCode: Long)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+
+
+@mainLayout(
+    pageTitle = messages("propertyRepresentation.addAnAgent.title"),
+    backUri = Some("#")
```

Comment:
Is this a TODO?

Reply:
good catch. No it is not.

Reply:
so a BACK link would reload the same page? If there isn't a back link, then shouldn't we use `None`? I am asking because I haven't done much frontend-y work. But I don't understand why `#` would be a back link